pkgs/applications/editors/rcptt/default.nix

```
@@ -0,0 +1,85 @@
+# Intende useage
+# Build the elexis-congo app via mvn -V clean verify -f elexis-congo/pom.xml
+# nix-build Elexis_Congo.nix
+# start one of the binaries under result/bin/
+#
+{ lib, pkgs, stdenv, bundlerEnv, ruby, makeWrapper, makeDesktopItem
+, gnome, git, gtk3, jdk11, fontconfig, freetype, glib
+, gsettings-desktop-schemas, webkitgtk, docutils, perl, xorg, zlib
+, libXrender, libsecret, unzip, rsync, fetchzip, ... }:
```

```suggestion
{ lib, pkgs, stdenv, bundlerEnv, ruby, makeWrapper, makeDesktopItem
, gnome, git, gtk3, jdk, fontconfig, freetype, glib
, gsettings-desktop-schemas, webkitgtk, docutils, perl, xorg, zlib
, libXrender, libsecret, unzip, rsync, fetchzip }:
```
There's no need to include `...` and `jdk` should be overridden in `all-packages.nix`.

Another suggestion would be to state inputs line-by-line like so:
```
{ lib
, sdtenv
, fetchGitHub
, ... # etc
```